README.rst

```diff
@@ -389,6 +389,29 @@ TMT_SHOW_TRACEBACK
     traceback is not printed out. When ``TMT_SHOW_TRACEBACK`` is
     set to any string except ``0``, traceback would be printed out.
 
+TMT_GIT_CREDENTIALS_URL\_, TMT_GIT_CREDENTIALS_VALUE\_
```

Comment:
Would it not be here better to include the suffix, as it is in the text?

```suggestion
TMT_GIT_CREDENTIALS_URL_<suffix>, TMT_GIT_CREDENTIALS_VALUE_<suffix>
```

Reply:
fixed in e7ac97ca